File: samples/python/parent.py
import time
import os
import syscall_lib

fd = -1 
counter = 0

def main():
    global fd, counter
    fd = syscall_lib.open()

    time.sleep(1)
    print("open MITOSIS client, fd {}", fd)

    syscall_lib.call_prepare(fd, 73)

    while True:
        time.sleep(1)
        counter += 1
        s = "check counter %d, fd %d" % (counter,fd)
        print(s)    

if __name__ == '__main__':
    main()

File: samples/cpp/parent.cc
#include <assert.h>
#include <unistd.h>
#include <stdio.h>
#include "../../mitosis-user-libs/mitosis-c-client/include/syscall.h"

int
main()
{
    int sd = sopen();
    assert(sd != 0);
    int cnt = 0;
    printf("time %d\n", cnt++);
    sleep(1);

    fork_prepare(sd, 73);

    while (1) {
        printf("time %d\n", cnt++);
        sleep(1);
    }
    return 0;
}
